assets/js/main.js

```diff
@@ -290,6 +290,32 @@ function elasticSearch(query, callback) {
 
 var searchSelectionId = 0;
 var searchListingLength = 0;
+function isNumeric(n) {
+    return !isNaN(parseFloat(n)) && isFinite(n);
+}
+function categoriseResult(post) {
+    if (!post.url_path || !post.url_path.raw) {
+        return ['home'];
+    }
+    var dir1 = post.url_path.raw.split('/')[1];
+    if (isNumeric(dir1)) {
+        var tags = ['post'];
+        if (post.title.raw.startsWith('Conversations with')) {
+            tags.push('conversations');
+        }
+        if (post.title.raw.startsWith('Reflections with')) {
+            tags.push('reflections');
+        }
+        if (post.title.raw.startsWith('Insights on')) {
+            tags.push('insights');
+        }
+        return tags;
+    }
+    var page_type = dir1.toLowerCase();
+        //.replace('-', ' ')
+        //.replace(/(^\w{1})|(\s+\w{1})/g, (letter) => letter.toUpperCase());
+    return ['page', page_type];
```

Comment:
Could this block be placed in an else clause to emphasise that this is if `dir1` is not numeric?